packages/cfd/src/Components/Routes/helpers.ts

```diff
@@ -0,0 +1,39 @@
+import { matchPath } from 'react-router';
+import { routes } from '@deriv/shared';
+import { TRouteConfig } from '../../types/common-prop-types';
+
+export const normalizePath = (path: string) => (/^\//.test(path) ? path : `/${path || ''}`); // Default to '/'
+
+export const findRouteByPath = (path: string, routes_config: TRouteConfig[]) => {
+    let result: TRouteConfig | undefined;
+
+    routes_config.some(route_info => {
+        let match_path;
```

Comment:
```suggestion
        let path_matched;
```

Reply:
@hamza-deriv @hirad-deriv @farzin-deriv we have the exact file in multiple packages, why not we get rid of this and use the one from `Shared` package?

Reply:
@hamza-deriv @hirad-deriv @farzin-deriv we have the exact file in multiple packages, why not we get rid of this and use the one from `Shared` package?

Reply:
We are going to refactor and remove routes from cfd package which is somehow out of the scope of this PR. We'll do the changes based on your suggestion in another PR since it requires the task of its own.